src/lib/components/inventory/selected_item_info.ts

```diff
@@ -130,6 +132,17 @@ export class SelectedItemInfo extends FloatElement {
         return html` <div class="container">${containerChildren}</div> `;
     }
 
+    renderFloatBar(): TemplateResult<1> {
+        if (!this.itemInfo) {
```

Comment:
I think we probably want to also prevent rendering the float bar unless the float is >0.

For example, some vanilla items with stickers applied on them will have a float of 0 from the API. Not a big deal though.